x-pack/plugins/observability/server/plugin.ts

```diff
@@ -73,7 +71,11 @@ interface PluginStart {
   alerting: PluginStartContract;
 }
 
-const sloRuleTypes = [SLO_BURN_RATE_RULE_TYPE_ID, OBSERVABILITY_THRESHOLD_RULE_TYPE_ID];
+const sloRuleTypes = [
+  SLO_BURN_RATE_RULE_TYPE_ID,
+  OBSERVABILITY_THRESHOLD_RULE_TYPE_ID,
+  ES_QUERY_ID,
```

Comment:
Not sure why we have ES_QUERY and OBSERVABILITY_THRESHOLD in `sloRuleTypes`, any idea?

Reply:
At first, I thought we will allow every o11y feature to create ES_QUERY and OBSERVABILITY_THRESHOLD. But It looks like that we only want user who have access to `LOGS` and `METRICS/INFRASTRUCTURE`. I will remove this rule type in teh SO for APM/SLO/UPTIME